lib/ansible/modules/apt.py

```diff
@@ -91,6 +91,16 @@
     type: bool
     default: 'no'
     version_added: "2.1"
+  allow_downgrade:
+    description:
+      - Corresponds to the C(--allow-downgrades) option for I(apt)
+      - This option enables the named package and version to replace an already installed higher version of that package.
+      - Note that setting allow_downgrade=True can make this module behave in a non-idempotent way.
+      - (The task could end up with a set of packages that does not match the complete list of specified packages to install).
+    aliases: [ allow-downgrade, allow_downgrades, allow-downgrades ]
```

Comment:
I'm not sure about adding aliases when no user has become familiar with anything and can learn it the correct way from the beginning.